dts/msm8226/msm8926-v2-720p-mtp.dts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	compatible = "qcom,msm8926-mtp", "qcom,msm8926";
+
+	qcom,msm-id = <200 0>;
+	qcom,board-id = <8 0>;
+
+	coolpad-8730l {
+		model = "Coolpad 8730L";
+		compatible = "coolpad,8730l", "qcom,msm8926", "lk2nd,device";
+	};
```

Comment:
Hm, maybe we should add some match rule here, in case some other 8926-mtp device appears later? i.e. match-cmdline or match-panel + a display panel list node